auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationContextConfigurationClient.java

```diff
@@ -196,6 +198,42 @@ private static AuthenticationConfiguration initializeConfiguration(final URI uri
         return configuration;
     }
 
+    /**
+     * Get all SSL contexts configured for this authentication context.
+     * This method is not part of the public API.
```

Comment:
The `org.wildfly.security.auth.client.AuthenticationContextConfigurationClient` class is public API so this method is also public API as discussed on Zulip. Should the sentence "This method is not part of the public API." be removed?

Reply:
@fjuma Yes this should be removed, good catch I removed it thank you!